docs/src/index.md

```diff
@@ -34,6 +34,7 @@ Modules = [
     ADAPT.Degenerate_ADAPT,
     ADAPT.TETRIS_ADAPT,
     ADAPT.ADAPT_QAOA,
+    ADAPT.ADAPT_QAOA.QAOApool,
```

Comment:
Add an s

(I am just exploring the interface here, don't mind me. I'm guessing this comment is going to have some link to the actual line of code?)